### ISSUE TITLE
Is there a typo in ReLU6 declaration ?

### ISSUE THREAD

**Issue by antoinecarme** [pytorch/pytorch#45120]
Was just doing some automated work on activations.  I see that ReLU6 is declared as a sub-class of Hardtanh. This smells a lot like a typo.

Not sure of the real impact. Seems harmless.

$  grep class nn/modules/activation.py

class Threshold(Module):
class ReLU(Module):
class RReLU(Module):
class Hardtanh(Module):
**_class ReLU6(Hardtanh):_**
class Sigmoid(Module):
class Hardsigmoid(Module):
class Tanh(Module):
class SiLU(Module):
class Hardswish(Module):
class ELU(Module):
class CELU(Module):
class SELU(Module):
class GLU(Module):
class GELU(Module):
class Hardshrink(Module):
class LeakyReLU(Module):
class LogSigmoid(Module):
class Softplus(Module):
class Softshrink(Module):
class MultiheadAttention(Module):
class PReLU(Module):
class Softsign(Module):
class Tanhshrink(Module):
class Softmin(Module):
class Softmax(Module):
class Softmax2d(Module):
class LogSoftmax(Module):

**Comment by ailzhang** [comment]
nope if you look at the documentation of `relu6` and `hardtanh` it's reasonable to do subclass. Closing since it's resolved.
